pkg/productinfo/alibaba/productinfo_alibaba.go

```diff
@@ -6,6 +6,7 @@ import (
 	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/log"
 	"github.com/aliyun/alibaba-cloud-sdk-go/services/ecs"
 	"github.com/banzaicloud/productinfo/pkg/productinfo"
+	"github.com/spf13/viper"
```

Comment:
I would recommend injecting configuration values and not accessing viper directly. See an example [here](https://github.com/banzaicloud/pipeline/blob/master/config/logger.go#L23)